src/repositories/channel.js

```diff
@@ -7,7 +7,7 @@ module.exports = [
         const get = async (opt) => {
             const channel;
             if(opt.hasOwnProperty('lean')){
-                channel = channelModel.findOne(opt.query).lean;
+                channel = channelModel.findOne(opt.query).lean();
```

Comment:
you can just pass in the value to it. You don't even have to check. If the value is false or undefined it just won't run the query as lean.

Reply:
okay

Reply:
well, feels like its more work than necessary to change every query function so it always contains a lean property

Reply:
But you don't need to add lean to every query function? You only need to add it to the get and not every call to get you need to set it to lean.

Reply:
I don't think you have to set that many queries to lean and its not any less work then attaching lean the options object every time if you are referring to calling the get function.